opencti-platform/opencti-front/src/private/components/data/ingestionCsv/IngestionCsvMapperTestDialog.tsx

```diff
@@ -113,7 +113,7 @@ const IngestionCsvMapperTestDialog: FunctionComponent<IngestionCsvMapperTestDial
         >
           <Button
             variant="contained"
-            color="secondary"
+            color={result?.test_mapper?.nbEntities ? 'primary' : 'secondary'}
```

Comment:
My comment was more on the creation form buttons, sorry for the misunderstanding.
the "Verify" button being the first main CTA, and then the "Create" button.

The verify button in the drawer form could be color "primary" when Create is disabled, and "secondary" otherwise.
This way user open the form, put the info, sees that Create is disabled, but "Verify" is primary color so it catches the eyes as the main CTA. Once verification is ok, and Create enabled, this is the new main CTA. Create is in primary, Verify  in Secondary.

Reply:
I understand your point.
So, can you confirm that the Test button from the Ingestion Csv Mapper Test Dialog stay unchanged ?

Reply:
yes it can stay unchanged IMO. The UI on this dialog is pretty clear I think.